deployment/win-x86/package.sh

```diff
@@ -1,9 +1,43 @@
 #!/usr/bin/env bash
-
+package_win32() (
+    local NSSM_URL="https://nssm.cc/ci/nssm-2.24-101-g897c7ad.zip"
+    local FFMPEG_URL="https://ffmpeg.zeranoe.com/builds/win64/static/ffmpeg-4.0.2-win32-static.zip"
```

Comment:
That's the only difference in two packages I see - `win32` vs `win64` in ffmpeg URL (and nssm path). I'd like to see either reduced copy-pasteness or maybe outright combined script with a parameter of what bitness to construct.

Reply:
agreed, I am not great at bash, I assume one could probably live in win-generic and get dot sourced (or whatever its called in bash) in like the common lib is.

Reply:
Agreed, we should add the `build-win` to the common functions file, and have it take in a 32/64-bit variable instead, to avoid the code duplication.